src/main/java/ch/uzh/ifi/hase/soprafs21/rest/mapper/UserDTOMapper.java

```diff
@@ -21,7 +21,7 @@
     @Mapping(source = "id", target = "id")
     @Mapping(source = "email", target = "email")
     @Mapping(source = "profilePicture", target = "profilePictureURL")
-    @Mapping(source = "latestLocation", target = "latestLocation")
+    //@Mapping(source = "latestLocation", target = "latestLocation")
```

Comment:
If we need this property (I think we do), we should add it to the entities & map it here. If we don't need it, we should remove it entirely.